varats/varats/tools/research_tools/vara.py

```diff
@@ -374,8 +374,58 @@ def verify_install(self, install_location: Path) -> bool:
         status_ok &= (install_location / "bin/opt").exists()
         status_ok &= (install_location / "bin/phasar-llvm").exists()
 
+        # Check that clang++ can display it's version
+
+        clang = local[str(install_location / "bin/clang++")]
+
+        (ret, stdout, _) = clang.run("--version")
```

Comment:
```suggestion
        # Check that clang++ can display it's version
        clang = local[str(install_location / "bin/clang++")]
        ret, stdout, _ = clang.run("--version")
```
Python does not require braces to unpack the return values